README.md

```diff
@@ -233,3 +233,6 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+## Contributor Code of Conduct
+Pandas has a strict code of conduct that all contributors and maintainers of this project must follow. More information can be found at: [Pandas Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

Comment:
Thanks for this @sparalic.

I think it makes more sense to have this as a paragraph of the `Contributing to pandas` section, not a section of its own.

Also, we don't capitalize the name `pandas`. Other than that, lgtm.

Reply:
@datapythonista thank you! I made the changes you suggested above